jobs/build_docker/build_docker.sh

```diff
@@ -17,17 +17,31 @@ on_imagebuilder_version=$( ./build-config/build-release-tools/HWIMO-BUILD ./buil
 --repo-dir ./$CLONE_DIR/on-imagebuilder \
 --is-official-release $IS_OFFICIAL_RELEASE )
 
-common_path=/var/renasar/on-http/static/http/common
-pxe_path=/var/renasar/on-tftp/static/tftp
 
 echo $SUDO_PASSWORD |sudo -S rm -rf /etc/apt/sources.list.d/rackhd.source.list
 echo "deb https://dl.bintray.com/$CI_BINTRAY_SUBJECT/debian trusty main" | sudo tee -a /etc/apt/sources.list.d/rackhd.source.list
 echo $SUDO_PASSWORD |sudo -S apt-get update
-echo $SUDO_PASSWORD |sudo -S apt-get --yes --force-yes remove on-imagebuilder
-echo $SUDO_PASSWORD |sudo -S apt-get --yes --force-yes install on-imagebuilder=$on_imagebuilder_version
+
```

Comment:
good idea. I've totally abandoned ```apt-get```  . so that we can run parallel docker-build on one slave .
please review again.
@PengTian0